### PR TITLE
Add a getting started section and some screenshots

### PR DESCRIPTION
**Pull request by pilhuhn**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -26,6 +28,45 @@ like circuit breakers or request rates.
...
+To deploy Kiali to your Istio-enabled OpenShift cluster you can run the following. Kiali currently requires Istio version 0.7.1.
+
+```
+export NAMESPACE=istio-system
+ 
+oc create -f https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-configmap.yaml -n ${NAMESPACE}
+curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali.yaml | \
+   IMAGE_NAME=kiali/kiali \
+   IMAGE_VERSION=latest \
+   NAMESPACE=${NAMESPACE} \
```
Comment (jmazzitelli) [inline index.md]: I would just hardcode istio-system here - don't make it seem like its optional and can be any value. AFAIK, we have to be in istio-system with the rest of istio

File: index.md [hunk 2]
```diff
@@ -26,6 +28,45 @@ like circuit breakers or request rates.
...
+
+```
+
+If you don't have `envsubst` installed, you can get it via the Gnu `gettext` package.
+
+Once the above has completed and the Docker image has been pulled from Dockerhub, go to the OpenShift console, select the istio-system project and determine the base-URL of Kiali
+
+![OpenShift console](/assets/img/os-console.png)
+
+In this case it is `http://kiali-istio-system.192.168.64.13.nip.io`.
```
Comment (jmazzitelli) [inline index.md]: you should tell them, "But in your network environment the IP address might be different than what you see here (192.168.64.13)".

File: index.md [hunk 3]
```diff
@@ -26,6 +28,45 @@ like circuit breakers or request rates.
...
+```
+export NAMESPACE=istio-system
+ 
+oc create -f https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali-configmap.yaml -n ${NAMESPACE}
+curl https://raw.githubusercontent.com/kiali/kiali/master/deploy/openshift/kiali.yaml | \
+   IMAGE_NAME=kiali/kiali \
+   IMAGE_VERSION=latest \
+   NAMESPACE=${NAMESPACE} \
+   VERBOSE_MODE=4 envsubst | oc create -n ${NAMESPACE} -f -
+
```
Comment (jmazzitelli) [inline index.md]: create the admin permissions, too.

**Review by jmazzitelli — CHANGES_REQUESTED** [review]

**Comment by pilhuhn** [comment]
cc @rettori

**Review by jmazzitelli — APPROVED** [review]

**Pull request merged**